packages/sfui/typography/package.json

```diff
@@ -22,7 +22,7 @@
     },
     "devDependencies": {
         "typescript": "^4.9.5",
-        "vite": "^4.1.2",
+        "vite": "^4.3.0",
```

Comment:
Not really needed in this PR, but good update! They did huge [performance boost in this `4.3.0` release ](https://vitejs.dev/blog/announcing-vite4-3.html)

Reply:
Indeed, and I needed as well to update vite in `apps/test/vue` because then my plugin did not match. Then I though I will unify it now as Im working with it